accounts-db/src/accounts.rs

```diff
@@ -669,6 +670,10 @@ impl Accounts {
         );
         self.accounts_db
             .store_cached_inline_update_index((slot, &accounts_to_store[..]), Some(&transactions));
+        accounts_to_store
+            .iter()
+            .map(|(_, account)| account.data().len() as u64)
```

Comment:
there is a fn maybe in append vec or accounts that tells you the total bytes, including owner, etc. This is only summing data len(), which is different than bytes stored.